valhalla/mjolnir/node_expander.h

```diff
@@ -217,6 +217,8 @@ struct Node {
   uint32_t end_of;
   // the graphid of the node
   baldr::GraphId graph_id;
+  // grid Id within the tile (used for spatial node sorting)
+  uint32_t grid_id;
```

Comment:
if we wanted to keep this completely hidden as an implementation detail we could even use the spare bits in graph_id rather add another field here

Reply:
it would certainly be less readable though..

Reply:
not sure we need to keep this hidden - especially if we add a mjolnir config for the grid sizing (that would make it pretty obvious!).

Reply:
yeah sorry i just mean hidden in the sense that its not part of the c++ api (methods we expose in header files). feel free to ignore this :smile: was just an errant thought!